docs/guides/algorithmiq-tem.ipynb

```diff
@@ -279,7 +281,7 @@
     "<LegacyContent>\n",
     "Name | Type | Description | Required | Default | Example\n",
     "-- | -- | -- | -- | -- | --\n",
-    "`pubs` | Iterable[EstimatorPubLike] | An iterable of PUB-like (primitive unified bloc) objects, such as tuples `(circuit, observables)` or `(circuit, observables, parameters, precision)`. See [Overview of PUBs](/docs/guides/primitive-input-output#overview-of-pubs) for more information. The circuits don't need to be ISA circuits. | Yes | N/A | (circuit, observables)\n",
+    "`pubs` | Iterable[EstimatorPubLike] | An iterable of PUB-like (primitive unified bloc) objects, such as tuples `(circuit, observables)` or `(circuit, observables, parameters, precision)`. See [Overview of PUBs](/guides/primitive-input-output#overview-of-pubs) for more information. If a non-ISA circuit is passed, it will be transpiled with optimal settings. If an ISA circuit is passed, it will not be transpiled, in this case the observable must be defined on the whole QPU. | Yes | N/A | (circuit, observables)\n",
     "`backend_name` | str | Name of the backend to make the query.| No | If not provided, the least busy backend will be used. | \"ibm_fez\"\n",
     "`instance` | str | The hub/group/project to use in that format. | Yes | N/A |\"hub1/group1/project1\"\n",
```

Comment:
```suggestion
```

Reply:
Also delete the instance row in the same table within the CloudContent tags below